src/features/machine/computer.js

```diff
@@ -47,20 +48,42 @@ export const transformShape = (data, vertex, amount, trackIndex=0, numLoops) =>
   let transformedVertex = vertex
 
   if (data.repeatEnabled && data.growEnabled) {
-    transformedVertex = scale(transformedVertex, 100.0 + (data.growValue * amount))
+    var growAmount = 100
```

Comment:
Super nitpicky, but I would try to stop using 'var' in favor of either 'let' or 'constant'